salt/modules/cmdmod.py

```diff
@@ -294,6 +294,9 @@ def _run(cmd,
     if runas is None and '__context__' in globals():
         runas = __context__.get('runas')
 
+    if password is None and '__context__' in globals():
+        password = __context__.get('runas_passwd')
```

Comment:
I think runas_password is better so that it isn't confusing where one is shortened and one isn't.